tests/JWT.Tests.NETFramework/DecodeTests.cs

```diff
@@ -1,78 +0,0 @@
-﻿using FluentAssertions;
-using JWT.Tests.Common;
-using JWT.Tests.NETFramework.Serializers;
-using Xunit;
-
-namespace JWT.Tests.NETFramework
-{
-    public class DecodeTests
-    {
-        [Fact]
-        public void Decode_Should_Decode_Token_To_Json_Encoded_String_With_WebScript_Serializer()
-        {
-            var serializer = new WebScriptJsonSerializer();
-            JsonWebToken.JsonSerializer = serializer;
```

Comment:
These tests are now missing. I think we should modify them to use an instance instead of the static `JsonWebToken` object.